app/uk/gov/hmrc/cataloguefrontend/connector/PlatformInitiativesConnector.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cataloguefrontend.connector
```

Comment:
Can we put all the PlatformInitiative code into a feature branch (see shuttering for example).
As well as finding the code more easily, (and could be moved out to it's own service if required), it can benefit from package visibility (e.g. package private domain) - note the ruby-on-rails play packaging style doesn't really fit java/scala - we rarely share private code across controllers, and if we did, it would be through inheritance.

Reply:
I've put all the code in its own feature branch now.